src/libraries/System.Private.Xml/src/System/Xml/Schema/ContentValidator.cs

```diff
@@ -876,7 +876,7 @@ sealed class RangeNode : InteriorNode {
 
         public override void Dump(StringBuilder bb, SymbolsDictionary symbols, Positions positions) {
             LeftChild.Dump(bb, symbols, positions);
-            bb.Append($"{{{Convert.ToString(min, NumberFormatInfo.InvariantInfo)}, {Convert.ToString(max, NumberFormatInfo.InvariantInfo)}}}");
+            bb.Append(NumberFormatInfo.InvariantInfo, $"{{{min}, {max}}");
```

Comment:
Did escaping of a trailing `}` become optional?
```suggestion
            bb.Append(NumberFormatInfo.InvariantInfo, $"{{{min}, {max}}}");
```

Reply:
> Did escaping of a trailing } become optional?

No :smile:  But this is dead code (behind a compilation constant that nothing it enabling) so nothing caught my typo.  Thanks.